hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationFuture.java

```diff
@@ -218,126 +169,76 @@ public E getSafely() {
     }
 
     @Override
-    public E get(long timeout, TimeUnit unit) throws InterruptedException, ExecutionException, TimeoutException {
-        Object unresolvedResponse = waitForResponse(timeout, unit);
-        return (E) resolveOrThrow(unresolvedResponse);
-    }
-
-    private Object waitForResponse(long time, TimeUnit unit) {
-        if (responseAvailable(response)) {
-            return response;
+    public E get() throws InterruptedException, ExecutionException {
```

Comment:
why not just delegate to `get(timeToEndOfTheUniverse, ..)` ? 
it's pretty much the same logic.